GiniSwitchSDK/GiniSwitchSDK/Classes/ExtractionCollection.swift

```diff
@@ -10,7 +10,7 @@ import UIKit
 
 public class ExtractionCollection:BaseApiResponse {
 
-    public let extractions:[Extraction]
+    public var extractions:[Extraction]
```

Comment:
"I'm trapped in an incremental PR and I don't know how to get the hell out of here"